src/SourceBuild/content/repo-projects/deployment-tools.proj

```diff
@@ -7,12 +7,19 @@
     <GlobalJsonFile>$(ProjectDirectory)global.json</GlobalJsonFile>
     <NuGetConfigFile>$(ProjectDirectory)NuGet.config</NuGetConfigFile>
     <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+
+    <PackageVersionPropsFlowType>DependenciesOnly</PackageVersionPropsFlowType>
   </PropertyGroup>
 
   <ItemGroup>
     <RepositoryReference Include="runtime" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Workaround missing source link version until arcade bootstrap version is updated -->
```

Comment:
Can you reference a tracking issue in this comment to help know when the workaround can be removed?

Reply:
Depending on how new the latest bootstrap is, it may be new enough already. I can check.

Reply:
Turns out it's new enough. I believe this means the workaround can generally be removed. I'll do that in a separate PR first.